src/App.jsx

```diff
@@ -1,45 +1,33 @@
-import React from 'react';
-import './App.scss';
+import React from "react";
+import "./App.scss";
+import { Person } from "./components/Person/Person";
 
 export const misha = {
-  name: 'Misha',
+  name: "Misha",
   age: 37,
-  sex: 'm',
+  sex: "m",
   isMarried: true,
-  partnerName: 'Natasha',
+  partnerName: "Natasha",
 };
 
 export const olya = {
-  name: 'Olya',
-  sex: 'f',
+  name: "Olya",
+  sex: "f",
   isMarried: true,
-  partnerName: 'Maksym',
+  partnerName: "Maksym",
 };
 
 export const alex = {
-  name: 'Alex',
+  name: "Alex",
   age: 25,
-  sex: 'm',
+  sex: "m",
   isMarried: false,
 };
```

Comment:
The `alex` object is missing the `partnerName` property. If the `Person` component expects this property, it might lead to errors. Consider adding a `partnerName` property, even if it's null or an empty string, to maintain consistency with other person objects.